package.json

```diff
@@ -17,16 +17,16 @@
   "license": "MIT",
   "engines" : { "node" : ">=4.1.0 <5.5.0" },
   "dependencies": {
-    "body-parser": "^1.6.5",
-    "express": "^4.8.5",
-    "twilio": "^1.6.0",
-    "morgan": "^1.2.3",
-    "jade": "1.7.x"
+    "body-parser": "^1.15.2",
+    "express": "^4.14.0",
+    "twilio": "3.0.0-rc.13",
```

Comment:
💯 for using a strict version.

The tutorials are an extension of Twilio's documentation -- https://www.twilio.com/docs/libraries/node#installation-nextgen.